ext/ddtrace.stub.php

```diff
@@ -139,6 +151,11 @@ class SpanData {
          */
         public array $peerServiceSources = [];
 
+        /**
+         * @var GitMetadata|null The git metadata of the span
+         */
+        public GitMetadata|null $gitMetadata = null;
+
```

Comment:
I think it should be on the trace local root span, i.e. RootSpanData only. No point in propagating.